smart-contracts/package.json

```diff
@@ -7,39 +7,39 @@
     "test": "test"
   },
   "engines": {
-    "node": "10.16.3"
+    "node": "10.17.0"
```

Comment:
Did we want to update Renovate to not touch the node version?

Reply:
I think @cnasc has recently submited a fix to only require any version `10`